### ISSUE TITLE
Support adding new file after compilation

### ISSUE THREAD

**Issue by jc4x4** [dtolnay/automod#22]
I noticed that if I add a new file after a build, the new file would not be included in the next run. I have to manually clean the build cache via `cargo clean`. Is there a better/automatic way of doing this?

**Comment by dtolnay** [comment]
I think if you add a build.rs containing the following (where `path/to/dir` is the relative path from the directory containing Cargo.toml to the directory containing the automatically included files) then it'll rebuild when a file is added:

```rust
fn main() {
    println!("cargo:rerun-if-changed=path/to/dir");
}
```

**Comment by jc4x4** [comment]
Right, this is also what I got after searching for a while. I can confirm it works well. It would be nice to call this out in the example in the README.md. Here is a PR: #23. 

i.e.
```
fn main() {
    println!("cargo:rerun-if-changed=tests/regression");
}
```
